.github/docker/Dockerfile.glibc

```
@@ -5,6 +5,7 @@ WORKDIR /mongodb-client-encryption
 COPY . .
 
 RUN node /mongodb-client-encryption/.github/scripts/libmongocrypt.mjs
```

```suggestion
RUN npm run install:libmongocrypt
```

Optional, let me know if you want to make this change otherwise, good to merge.